helpers/homework.js

```diff
@@ -1,23 +1,14 @@
 const _ = require('lodash');
 
-function isGraded(submission) {
-	return typeof submission.grade === 'number' || !!submission.gradeComment || !_.isEmpty(submission.gradeFileIds);
-}
-
 function getGradingFileName(file) {
```

Comment:
The function is still alive for backward compatibility? 
Sound like a seach all used place and replace it with file.name.